tests/jsunit/variable_model_test.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Blockly Tests
```

Comment:
Interesting--this may be an argument for pushing updated versions of blockly_uncompressed when we add new files, even if they're only added to develop.  @RoboErikG we can discuss this later--it's not a blocker for this PR.